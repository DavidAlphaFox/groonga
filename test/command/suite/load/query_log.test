table_create Numbers TABLE_PAT_KEY Int32

#@collect-query-log true
load --table Numbers
[
{"_key": 1},
{"_key": "string"},
{"_key": 2}
]
#@collect-query-log false
