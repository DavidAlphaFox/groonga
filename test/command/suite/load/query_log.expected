table_create Numbers TABLE_PAT_KEY Int32
[[0,0.0,0.0],true]
load --table Numbers
[
{"_key": 1},
{"_key": "string"},
{"_key": 2}
]
[[[-22,0.0,0.0],"<Numbers>: failed to cast to <Int32>: <\"string\">"],2]
#|e| <Numbers>: failed to cast to <Int32>: <"string">
#>load --table "Numbers"
#:000000000000000 load(2)
#<000000000000000 rc=-22
